src/MicrosoftTeams.schema.json

```diff
@@ -35,17 +35,20 @@
                 "websiteUrl": {
                     "type": "string",
                     "description": "The url to the page that provides support information for the app.",
-                    "maxLength": 2048
+                    "maxLength": 2048,
+                    "pattern": "^[Hh][Tt][Tt][Pp][Ss]://"
```

Comment:
We don't ever load any of these URLs in our iframes; are you sure that they need to be https?